packages/cadl-python/test/generated/interfaces/multiinterfaceclient/aio/operations/_operations.py

```diff
@@ -1,344 +0,0 @@
-# pylint: disable=too-many-lines
-# coding=utf-8
```

Comment:
why are these files deleted?

Reply:
the _model_base.py file still exists for this folder, so it should either be fully deleted or these files hould still exist

Reply:
The `interfaces/multiinterfaceclient` still exists.
Some previous PR changed this package from non-mixin to mixin client, so the folder `operations` should be renamed to `_operations`.

But it turned out that folder `_operations` was added and `operations` was leaved with no change. So here is just deleting the `operations` and `aio/operations` folders.

Reply:
@changlong-liu @iscai-msft It is caused by https://github.com/Azure/autorest.python/issues/1409. Since we used `solution 2`, the issue still exists for `cadl-python`

Reply:
The `--clear-output-folder` is not supported in `cadl-python` because the `cadl-python` has gotten ride of `autorest`?
Maybe we should create issue to add this feature for cadl generation.

Reply:
Yeah, I think we can make feature request for cadl since all the language need similar flag to remove outdated code